### PR TITLE
ci: Update version bump workflow to automate changelog generation

### PR DESCRIPTION
**Pull request by edgarrmondragon**
- Update changelog using commitizen
- Create draft release

### CODE REVIEW

File: .github/workflows/version_bump.yml [hunk 1]
```diff
@@ -45,27 +46,31 @@ jobs:
...
         pipx install changelog-cli
 
-    - name: Prepare Changelog
-      if: always() && (github.event.inputs.prerelease == 'none')
-      run: |
-        bump_lowercase=$(echo "${{ github.event.inputs.bump }}" | tr '[:upper:]' '[:lower:]')
-        changelog release --$bump_lowercase --yes
+    - name: Draft Release
+      id: draft-release
+      uses: softprops/action-gh-release@v1
```
[the remark this answers is not in the document]
Comment (edgarrmondragon) [reply .github/workflows/version_bump.yml:66]: Good catch. I wonder why it still works then 🤔

**Comment by edgarrmondragon** [comment]
Example release draft: https://github.com/meltano/meltano/releases/tag/untagged-3e7e6f304c68f7679983

Screen Shot 2022-07-14 at 19 14 20

Example changelog update: https://github.com/meltano/meltano/blob/714722354945a25b08094a4b2c54685b773f68ef/CHANGELOG.md

Screen Shot 2022-07-14 at 19 14 50

File: pyproject.toml [hunk 2]
```diff
@@ -133,6 +133,24 @@ version_files = [
     'docs/package.json:^  "version":',
 ]
 
+[tool.commitizen.customize]
+commit_parser = '^(?P<change_type>feat|fix|refactor|perf|BREAKING CHANGE|docs)(?:\((?P<scope>[^()\r\n]*)\)|\()?(?P<breaking>!)?:\s(?P<message>.*)?'
+change_type_order = [
+  "BREAKING CHANGE",
```
Comment (WillDaSilva) [inline pyproject.toml:139]:
```suggestion
  "BREAKING CHANGES",
```

File: pyproject.toml [hunk 3]
```diff
@@ -133,6 +133,24 @@ version_files = [
...
+  "✨ New",
+  "🐛 Fixes",
+  "⚙️ Under the Hood",
+  "⚡ Performance",
```
Comment (WillDaSilva) [inline pyproject.toml:143]: For consistency with "Documentation Improvements":

```suggestion
  "⚡ Performance Improvements",
```

File: pyproject.toml [hunk 4]
```diff
@@ -133,6 +133,24 @@ version_files = [
...
+  "📚 Documentation Improvements",
+]
+
+[tool.commitizen.customize.change_type_map]
+feat = "✨ New"
+fix = "🐛 Fixes"
+refactor = "⚙️ Under the Hood"
+docs = "📚 Documentation Improvements"
+perf = "⚡ Performance"
```
Comment (WillDaSilva) [inline pyproject.toml:152]:
```suggestion
perf = "⚡ Performance Improvements"
```

**Review by WillDaSilva — APPROVED** [review]

**Review by tayloramurphy — APPROVED** [review]
LGTM - I'll likely start updating the actual release notes once its published - but this is a great place to start!

**Pull request merged**